src/consts.js

```diff
@@ -26,12 +26,12 @@ module.exports = {
         dockEventHidden: "dock.event.hidden",
         dockEventIconSet: "dock.event.icon.set",
         dockEventShown: "dock.event.shown",
-        globalShortcutCmdRegister: "global.shortcut.cmd.register",
-        globalShortcutCmdIsRegistered: "global.shortcut.cmd.is.register",
-        globalShortcutCmdUnregister: "global.shortcut.cmd.unregister",
-        globalShortcutCmdUnregisterAll: "global.shortcut.cmd.unregister.all",
-        globalShortcutEventProcessFinished: "global.shortcut.event.process.finished",
-        globalShortcutEventTriggered: "global.shortcut.event.triggered",
+        globalShortcutEventRegistered: "global.shortcuts.event.registered",
+        globalShortcutEventIsRegistered: "global.shortcuts.event.is.registered",
+        globalShortcutEventUnregistered: "global.shortcuts.event.unregistered",
+        globalShortcutEventUnregisteredAll: "global.shortcuts.event.unregistered.all",
+        globalShortcutEventProcessFinished: "global.shortcuts.event.process.finished",
+        globalShortcutEventTriggered: "global.shortcuts.event.triggered",
```

Comment:
We misunderstood each other: could you rollback all `globalShortcutCmd...` events but replace the `globalShortcutEventProcessFinished` with:

- global.shortcuts.event.
- global.shortcuts.event.is.
- global.shortcuts.event.
- global.shortcuts.event.unregistered.all

Also, could you make sure there's an `s` at the end of `globalShortcut` in the key?

This should look like this:

```javascript
globalShortcutsCmdRegister: "global.shortcuts.cmd.register",
globalShortcutsCmdIsRegistered: "global.shortcuts.cmd.is.register",
globalShortcutsCmdUnregister: "global.shortcuts.cmd.unregister",
globalShortcutsCmdUnregisterAll: "global.shortcuts.cmd.unregister.all",
globalShortcutsEventRegistered: "global.shortcuts.event.registered",
globalShortcutsEventIsRegistered: "global.shortcuts.event.is.registered",
globalShortcutsEventUnregistered: "global.shortcuts.event.unregistered",
globalShortcutsEventUnregisteredAll: "global.shortcuts.event.unregistered.all",
globalShortcutsEventProcessFinished: "global.shortcuts.event.process.finished",
globalShortcutsEventTriggered: "global.shortcuts.event.triggered",
```